src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color $transition-duration ease-in-out,
```

Comment:
As for me, just 'ease' would be enough, but this works so that doen't really matter))